.tekton/determine-image-digest.yaml

```
@@ -0,0 +1,37 @@
+apiVersion: tekton.dev/v1
+kind: Task
+metadata:
+  name: determine-image-digest
+  namespace: rh-acs-tenant
+spec:
+  description: Determines image digests.
+  params:
+  - name: IMAGE_REPOSITORY
+    description: Image repository.
+    type: string
+  - name: IMAGE_TAG
+    description: Image tag.
+    type: string
+  results:
+  - name: IMAGE_DIGEST
+    description: Image digest.
```

It may help to briefly mention the expected format for future generations. Knowing this helps not to worry manually prepending `sha256:`.
```suggestion
    description: Image digest in the format `sha256:abcdef0123`.
```